Bug: https://bugs.gentoo.org/858833

From: orbea <orbea@riseup.net>
Date: Tue, 19 Jul 2022 07:46:36 -0700
Subject: [PATCH] build: Fix the build with slibtool

Adding LT_INIT to configure.in generates the libtool script in the build
directory which is required by rlibtool to determine if the build is
shared or static.

--- a/configure.in
+++ b/configure.in
@@ -398,7 +398,7 @@ AC_PATH_PROG(RSYNC, rsync)
 AC_PATH_PROG(SVN, svn)
 AC_PROG_AWK
 AC_PROG_LN_S
-AC_CHECK_TOOL(RANLIB, ranlib, true)
+LT_INIT
 dnl AC_PATH_PROG(PERL_PATH, perl)
 AC_CHECK_PROGS(LYNX_PATH,[lynx links elinks], [lynx])
 
